spec/features/delete_taxon_spec.rb

```diff
@@ -0,0 +1,142 @@
+require "rails_helper"
+
+RSpec.feature "Delete Taxon", type: :feature do
+  include ContentItemHelper
+  include PublishingApiHelper
+
+  scenario "a taxon with no children" do
+    given_a_taxon_with_no_children
+    when_i_visit_the_taxon_page
+    and_i_click_to_delete_the_taxon
+    then_i_expect_to_be_informed_that_im_about_to_delete_the_taxon
+    and_i_should_see_the_action_buttons
+    and_i_click_delete
```

Comment:
What do you think of renaming this to `when_i_click_delete`, as it's an action we are performing rather than an assertion?